spec/fueling_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'fueling'
+
+RSpec.describe Fueling do
+  it 'uses row_id as its primary key' do
+    fueling = create :fueling
+    expect(fueling.row_id).to eq(fueling.id)
+  end
+
+  it 'reverse-sorts by fuel date by default' do
+    f1 = create :fueling, ftk_date: 1.day.ago
+    f2 = create :fueling, ftk_date: 3.days.ago
+    f3 = create :fueling, ftk_date: 4.days.ago
+    f4 = create :fueling, ftk_date: 2.days.ago
+    expect(Fueling.pluck(:row_id)).to eq([f1, f4, f2, f3].map(&:id))
```

Comment:
Why don't you instead create them in order, so you can write f1, f2, f3 etc, instead of having to, while you're writing your array, look above to see when the ftk_date is? That'll make it more readable.
alternatively, just do the sorting method in the test that the test is describing. reverse_sort them by date. It's okay in this case because you're not testing sorting by date, you're testing that your method sorts by date.